go/tasks/plugins/k8s/sagemaker/utils.go

```diff
@@ -0,0 +1,281 @@
+package sagemaker
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/lyft/flytestdlib/logger"
+
+	"github.com/Masterminds/semver"
+	commonv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/common"
+	awssagemaker "github.com/aws/amazon-sagemaker-operator-for-k8s/controllers/controllertest"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+	sagemakerSpec "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins/sagemaker"
+	"github.com/lyft/flyteplugins/go/tasks/plugins/k8s/sagemaker/config"
+	"github.com/pkg/errors"
+
+	"github.com/golang/protobuf/proto"
+)
+
+func getAPIHyperParameterTuningJobStrategyType(
```

Comment:
if we just typecast you would not need this. And that should be ok right?

Reply:
all the switch statements below?

Reply:
how do you type cast from 

```go
const (
	HyperparameterTuningStrategy_BAYESIAN HyperparameterTuningStrategy_Value = 0
	HyperparameterTuningStrategy_RANDOM   HyperparameterTuningStrategy_Value = 1
)
```

to 
 ```go
const (
	BayesianSageMakerAPIHyperParameterTuningJobStrategyType commonv1.HyperParameterTuningJobStrategyType = "Bayesian"
	RandomSageMakerAPIHyperParameterTuningJobStrategyType   commonv1.HyperParameterTuningJobStrategyType = "Random"
)
```
?

Reply:
@kumare3 ^

Reply:
oh I saw your code snippet above

Reply:
wait that doesn't work right?